src/Login/Login.jsx

```diff
@@ -36,6 +45,14 @@ class Login extends React.Component {
     this.state = getInitialState(window.innerHeight - 221);
     this.resetState = this.resetState.bind(this);
     this.updateDimensions = this.updateDimensions.bind(this);
+
+    // using objects for autoCompleteRefs, suggestionLists and
+    // currentLoginUrl because we need one for each login dropdown
+    this.autoCompleteRefs = {};
+    this.state = {
```

Comment:
add `height: getInitialState(window.innerHeight - 221),` into this block? This should fix the DNA image display issue

Reply:
thank you 😭